images/marketplace-operator.yml

```diff
@@ -8,7 +8,7 @@ content:
     git:
       branch:
         target: release-{MAJOR}.{MINOR}
-      url: git@github.com:operator-framework/operator-marketplace.git
+      url: git@github.com:openshift-priv/operator-marketplace.git
```

Comment:
same issue as 4.5 here... the priv repo is broken, leave alone until it's working. and when it is we'll need a public_upstream just for this.